src/main/diagnostics/connectedToInternet.ts

```diff
@@ -2,19 +2,45 @@ import fetch from 'node-fetch';
 
 import { DiagnosticsCategory, DiagnosticsChecker } from './types';
 
+import Logging from '@/utils/logging';
+
+const console = Logging.diagnostics;
+
+// Returns the timeout, in milliseconds, for the network connectivity check.
+function getTimeout(): number {
+  if (process.env.RD_CONNECTED_TO_INTERNET_TIMEOUT) {
+    const parsedTimeout = parseInt(process.env.RD_CONNECTED_TO_INTERNET_TIMEOUT);
+
+    if (!isNaN(parsedTimeout) && parsedTimeout > 0) {
```

Comment:
All comparison operators return `false` if one of their operands is `NaN`, so the extra check is not necessary:

```suggestion
    if (parsedTimeout > 0) {
```